.vitepress/theme/components/Beta.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <Badge
+    type="warning"
+    text="beta"
+    title="This is a beta feature. Beta features aren't part of the officially delivered scope that SAP guarantees for future releases."
```

Comment:
```suggestion
    title="Beta features are planned to be generally available in subsequent releases, however, APIs and their behavior are not final and may change in the general release."
```
This is what we said in the glossary so maybe we can keep it the same?